dir.props

```diff
@@ -144,6 +144,12 @@
     -->
     <SkipBuildPackages>true</SkipBuildPackages>
   </PropertyGroup>
+  
+  <PropertyGroup>
+    <!-- By default make all libraries to be AnyCPU but individual projects can override it if they need to -->
+    <Platform>AnyCPU</Platform>
+    <OutputType>Library</OutputType>
+  </PropertyGroup>
```

Comment:
Yes we should remove them from the other csproj's as well. I will try to get those cleaned up with the larger pass over the repo we are working on now, but if they don't then we should definitely get them cleaned up later.